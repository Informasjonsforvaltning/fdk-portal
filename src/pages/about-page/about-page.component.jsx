import React from 'react';
import DocumentMeta from 'react-document-meta';
import { Link } from 'react-router-dom';

import localization from '../../lib/localization';

export const AboutPage = () => {
  const meta = {
    title: `Om ${localization.about.title}`,
    description: localization.about.ingress
  };
  return (
    <div className="container">
      <DocumentMeta {...meta} />
      <div className="row">
        <div className="col-lg-8 offset-lg-2">
          <h1 className="title">{localization.about.title}</h1>
          <div className="mb-2">
            <p className="fdk-ingress">{localization.about.titleSub}</p>
            <p className="fdk-ingress">{localization.about.ingress}</p>
          </div>
          <div className="fdk-textregular">
            <p
              dangerouslySetInnerHTML={{ __html: localization.about.maintext }}
            />
            <p
              dangerouslySetInnerHTML={{
                __html: localization.about.dataNorgeText
              }}
            />
            <p>
              <b>{localization.about.register}</b>
              <br />
              <Link to="/about-registration">
                {localization.about.helpToRegister}
              </Link>
            </p>
            <p>
              <a
                title="Lenke til registreringslÃ¸sning"
                target="_blank"
                rel="noopener noreferrer"
                href="https://registrering.fellesdatakatalog.brreg.no/"
              >
                <span className="fdk-button fdk-button-default">
                  Kom i gang med registreringen
                </span>
              </a>
            </p>
          </div>
        </div>
      </div>
    </div>
  );
};
