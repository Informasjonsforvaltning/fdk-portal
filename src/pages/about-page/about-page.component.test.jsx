import React from 'react';
import { shallow } from 'enzyme';
import { AboutPage } from './about-page.component';

test('should render AboutPage correctly', () => {
  const wrapper = shallow(<AboutPage />);
  expect(wrapper).toMatchSnapshot();
});
