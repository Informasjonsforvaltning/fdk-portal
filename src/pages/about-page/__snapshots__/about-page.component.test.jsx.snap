// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AboutPage correctly 1`] = `
<div
  className="container"
>
  <SideEffect(DocumentMeta)
    description="The National Data Catalog is an overview showing which data the various public agencies have registered, how they are connected and what they mean. The National Data Catalog makes it possible for you to search in the information registered with the public agencies and use the information for analytic purposes."
    title="Om The National Data Catalog"
  />
  <div
    className="row"
  >
    <div
      className="col-lg-8 offset-lg-2"
    >
      <h1
        className="title"
      >
        The National Data Catalog
      </h1>
      <div
        className="mb-2"
      >
        <p
          className="fdk-ingress"
        >
          

        </p>
        <p
          className="fdk-ingress"
        >
          The National Data Catalog is an overview showing which data the various public agencies have registered, how they are connected and what they mean. The National Data Catalog makes it possible for you to search in the information registered with the public agencies and use the information for analytic purposes.
        </p>
      </div>
      <div
        className="fdk-textregular"
      >
        <p
          dangerouslySetInnerHTML={
            Object {
              "__html": "In Digital Agenda, the government states that the users should only have to provide public administration with the same information once. The various agencies within the administration must then share the necessary information with each other. The National Data Catalog is the first step in order to reach this goal. <br><br>The cooperative body SKATE (Managing and coordinating services in electronic administration) has described the ambition for information management as a process of three levels. The National Data Catalog being the first level, providing a common overview of data and a shared understanding of information in the shape of terms. The next level focuses on the availability of information in a common API Catalogue, this work is initiated. In the third level, everything should be set for reuse of data being the main rule and for the use of the principle of reporting information once only.<br><br>The Government's circular on digitalisation emphasizes that all public agencies must have an overview of which data they process, what the data mean, what they can be used for, which processes they are included in and who are allowed to use the data (information management).<br><br>In cooperation with public agencies, The Agency for Public Management and eGovernment (Difi) will establish a common framework for information management in public agencies. The National Data Catalog is a project to achieve common goals. Each agency must create its own overviews and descriptions, 'Order in house'. Difi is facilitating so that a common set of methods, standards and guidances is created, tested and used by agencies who is creating 'Order in house'. This includes organising and roles to establish systematic administration and good exploitation of data. An open network will inspire, inform and involve.<br><br> The framework  will help the political goals of 'once only' and increased use of public data. For more details, go to <a href='http://difi.no/informasjonsforvaltning'>http://difi.no/informasjonsforvaltning</a> <br><br>For further information about felles datakatalog see <a href='https://informasjonsforvaltning.github.io'>http://informasjonsforvaltning.github.io</a>",
            }
          }
        />
        <p
          dangerouslySetInnerHTML={
            Object {
              "__html": "<b>Import datasets from data.norge.no, geonorge.no with more</b><br>The National Data Catalog imports dataset descriptions from other catalogs, for instance <a href='https://data.norge.no' target='blank'>data.norge.no</a> (<a href='https://data.norge.no/data/direktoratet-forvaltning-og-ikt/datanorgeno-nasjonal-katalog-over-åpne-offentlige-datasett' target='blank'>API</a>) and geonorge.no. This is done by using the DCAT-AP-NO standard as a basis to exchange dataset descriptions.",
            }
          }
        />
        <p>
          <b>
            Registration
          </b>
          <br />
          <Link
            replace={false}
            to="/about-registration"
          >
            User manual to start registration (only in Norwegian)
          </Link>
        </p>
        <p>
          <a
            href="https://registrering.fellesdatakatalog.brreg.no/"
            rel="noopener noreferrer"
            target="_blank"
            title="Lenke til registreringsløsning"
          >
            <span
              className="fdk-button fdk-button-default"
            >
              Kom i gang med registreringen
            </span>
          </a>
        </p>
      </div>
    </div>
  </div>
</div>
`;
